packages/react-dom/src/client/ReactDOMFiberSelect.js

```diff
@@ -137,6 +137,7 @@ function updateOptions(
 export function getHostProps(element: Element, props: Object) {
   return Object.assign({}, props, {
     value: undefined,
+    defaultValue: undefined,
```

Comment:
Why do we do this?

Reply:
Since defaultValue is no longer a reserved word, it prevents it from being assigned as an attribute.

Reply:
`defaultChecked` is not reserved either now. Why doesn't it need to be here?